src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitedObj = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
+
+  const resultObj = {};
+
+  splitedObj.forEach((el) => {
+    const splitedEl = el.split(':');
+    const right = splitedEl[1].trim();
+    const left = splitedEl[0].trim();
+
+    resultObj[left] = right;
+  });
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. These methods return a new array, so you should use it. Consider using `reduce` instead of `forEach` to directly construct the resulting object without mutating an external variable.